lib/routes/qingting/podcast.ts

```diff
@@ -29,75 +34,93 @@ export const route: Route = {
     description: `获取的播放 URL 有效期只有 1 天，需要开启播客 APP 的自动下载功能。`,
 };
 
+function getMediaUrl(channelId: string, midiaId: string) {
+    const path = `/audiostream/redirect/${channelId}/${midiaId}?access_token=&device_id=MOBILESITE&qingting_id=${qingtingId}&t=${Date.now()}`;
```

Comment:
The enclosure url of the [first item](https://github.com/DIYgod/RSSHub/pull/15251#issuecomment-2058407281) has `undefined` in plain string instead of an empty string.
```xml
<item>
<title>发刊词 | 纵览世界风云变幻，领略经世致用智慧</title>
<description/>
<link>https://www.qingting.fm/channels/293411/programs/11569182/</link>
<guid isPermaLink="false">https://www.qingting.fm/channels/293411/programs/11569182/</guid>
<pubDate>Mon, 14 Oct 2019 11:14:52 GMT</pubDate>
<itunes:image href="http://pic.qtfm.cn/channel/2019/04/12/9873ee0c204ee3522c4258f539bf5d4b.jpg!400"/>
<enclosure url="https://audio.qingting.fm/audiostream/redirect/293411/11569182?access_token=&device_id=MOBILESITE&qingting_id=undefined&t=1713281212586&sign=1b15f08fcd58a415cf609b7bb8e313ba" type="audio/x-m4a"/>
<itunes:duration>0:01:36</itunes:duration>
</item>
```

Reply:
Thanks for your review!